README.md

```diff
@@ -248,6 +248,7 @@ These are currently used to compute properties of a minimum energy conformation
 |`OpenFF Gen2 Hessian Dataset Protomers v1.0` | [2024-10-07-OpenFF-Gen2-Hessian-Dataset-Protomers-v1.0](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-10-07-OpenFF-Gen2-Hessian-Dataset-Protomers-v1.0/) | Hessian single points for the final molecules in the `OpenFF Gen2 Optimization Dataset Protomers v1.0` [dataset](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2021-12-21-OpenFF-Gen2-Optimization-Set-Protomers) | 'H', 'C', 'Cl', 'P', 'F', 'Br', 'O', 'N', 'S'||
 | `MLPepper-RECAP-Optimized-Fragments-Add-Iodines-v1.0` | [2024-10-11-MLPepper-RECAP-Optimized-Fragments-Add-Iodines-v1.0](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-10-11-MLPepper-RECAP-Optimized-Fragments-Add-Iodines-v1.0)  | Set of diverse iodine containing molecules with a number of calculated electrostatic properties.   | Br, Cl, S, B, O, Si, C, N, I, P, H, F|   |
 | `OpenFF Iodine Chemistry Hessian Dataset v1.0` | [2024-11-11-OpenFF-Iodine-Chemistry-Hessian-Dataset-v1.0](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-11-11-OpenFF-Iodine-Chemistry-Hessian-Dataset-v1.0) | Hessian single points for the final molecules in the `OpenFF Iodine Chemistry Optimization Dataset v1.0` [dataset](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2022-07-27-OpenFF-iodine-optimization-set) |  I, F, Br, C, Cl, O, S, N, H ||
+| `OpenFF Aniline Para Hessian v1.1` | [2024-11-12-OpenFF-Aniline-Para-Hessian-v1.1](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-11-12-OpenFF-Aniline-Para-Hessian-v1.1) | Hessian single points for the final molecules in the `OpenFF Aniline Para Opt v1.0` [dataset](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2021-04-02-OpenFF-Aniline-Para-Opt-v1.0), re-generated to preserve molecule IDs between opt and basic datasets. | 'O', 'Cl', 'S', 'Br', 'H', 'F', 'N', 'C' ||
```

Comment:
I think I put my v1.1 right below the v1.0 entry instead of at the end of the table. That seems slightly better for discoverability of the new version, but I don't have really strong feelings about this.